### PR TITLE
support json tag

### PR DESCRIPTION
**Pull request by wangqj**

### CODE REVIEW

File: model_builder_test.go [hunk 1]
```diff
@@ -140,8 +140,33 @@ func TestKeyFrom(t *testing.T) {
...
 			t.Errorf("get %v but not %v", tp, typed[i])
 		}
 	}
-}
\ No newline at end of file
+}
+
+func TestJsonTag(t *testing.T) {
+	type X struct {
+		A int
```
Comment (tianxiaoliang) [inline model_builder_test.go:151]: add ignore syntax “-”  for UT

**Review by tianxiaoliang — CHANGES_REQUESTED** [review]

**Comment by tianxiaoliang** [comment]
use other tag not json tag

**Comment by wangqj** [comment]
done

File: model_builder.go [hunk 2]
```diff
@@ -135,10 +142,26 @@ func (b modelBuilder) keyFrom(st reflect.Type) string {
...
 	if len(modelName) == 0 {
 		return false
 	}
 	return strings.Contains("uint uint8 uint16 uint32 uint64 int int8 int16 int32 int64 float32 float64 bool string byte rune time.Time", modelName)
 }
+
+// jsonNameOfField returns the name of the field as it should appear in JSON format
+// An empty string indicates that this field is not part of the JSON representation
+func (b modelBuilder) jsonNameOfField(field reflect.StructField) string {
+	if jsonTag := field.Tag.Get("json"); jsonTag != "" {
```
Comment (tianxiaoliang) [inline model_builder.go]: change json to swagger tag

**Review by tianxiaoliang — CHANGES_REQUESTED** [review]

**Pull request merged**